cppcache/src/ThinClientRegion.cpp

```diff
@@ -846,7 +847,8 @@ bool ThinClientRegion::containsValueForKey_remote(
 
   auto rptr = CacheableBoolean::create(ret);
 
-  rptr = std::dynamic_pointer_cast<CacheableBoolean>(handleReplay(err, rptr));
+  // rptr = std::dynamic_pointer_cast<CacheableBoolean>(handleReplay(err,
```

Comment:
If this is truly not needed, just deleted it.  Same for commented code above on line 802